lane_detection/src/lane_detector_classical (copy).py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
Remove files that don't belong to this branch from the branch?

Reply:
not exactly sure how, i think i've gone too far with these edits. when i try to remove them from this branch and then push, the deletions get reflected in master, which we don't want. i agree this is kinda sloppy but just for this time can we push it thru? it's only the python scripts that we don't really use, and then some minor edits on the plotting scripts for SVD stuff

Reply:
I actually think we should include svd testing files to `master` branch, as long as we documented down what the files are for eventually. wouldn't spend time removing those files either, since we are not pushing it for a production or whatsoever.

Reply:
i don't know. if a file like this one already exists, i can delete...

Reply:
yea can you `git rm ` this file and push another commit? 'cause I don't think this file is being used anywhere else, and I locally removed this file on my branch too. Let's not put this file in.

Reply:
i think i deleted it